### DISCUSSION TITLE
someIP/TP-packets arrival window of 5 second's hardcoding

### DISCUSSION THREAD

**Discussion by vidz12** [COVESA/vsomeip#700] [Q&A]
Reference: https://github.com/COVESA/vsomeip/blob/master/implementation/endpoints/src/tp_reassembler.cpp
Line no: 130

We see a timing constraint set for SOME/IP TP packets - "The fragmented packets should reach destination within 5000 ms" 

When we tried transmitting 1MB of UDP data using someIP-TP, sometimes receiver fails to receive complete packets (it is getting timed out due to 5seconds constraint) sent over event notify mechanism. 

Would like to understand how 5seconds window arrived to expect the packets to reach destination?

**Comment by KudoWu** [comment]
Hello, vidz12!

I would like to ask for some guidance on how to use someip-tp. I have already read the vsomeipUserguide and configured the settings as follows:

```
...
"someip-tp": {                                                                                                                 
    "client-to-service": ["0x80e9"]              
}, 
...
"max-payload-size-reliable": 1000000
```

However, I'm still encountering the following issue during actual use: `sei::send_intern: Dropping too big message (733962 Bytes). Maximum allowed message size is: 1416 Bytes`. 

Could you give your example or help me? Thank you very much.
